OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -3909,14 +3905,15 @@ embraces the base of the cross which is flanked by
                                       Double circles decorate the edges of the boards.
                                     </decoNote>
                                     <decoNote xml:id="b4" type="Endbands" color="brown">
-                                      <term key="slitBraid">Slit-braid</term> endbands are present.
+                                      <term key="slitBraid">Slit-braid</term> brown endbands are present.
                                     </decoNote>
-                                <decoNote xml:id="b5" type="SewingStations">4</decoNote>
-                                <decoNote xml:id="b6">
-                                  Chainstitch sewing on four sewing stations (two pairs); periodic fold pattern. Two S-plied threads are used.
-                                  The sewing starts from the first quire where, for each pair of sewing stations, there is one length of thread in the centre-fold,
-                                  while in the rest of the quires there are two.
-                                </decoNote>
+                                    <decoNote xml:id="b5" type="Sewing">
+                                      Chainstitch sewing on four sewing stations (two pairs); periodic fold pattern. Two S-plied threads are used.
+                                      The sewing starts from the first quire where, for each pair of sewing stations, there is one length of thread in the centre-fold,
+                                      while in the rest of the quires there are two.
```

Comment:
Two threads or two lengths of thread? If it's "two threads", then maybe this can simply be said explicitly